### DISCUSSION TITLE
Xarray 2021 User Survey

### DISCUSSION THREAD

**Discussion by jhamman** [pydata/xarray#4772] [Show and tell]
Hi All,

The Xarray developer team has put together a short user survey: https://forms.gle/TbixvHqY7VB3JERE6.

If you can spare ~5 minutes to fill it out, we’d appreciate it very much.

Thanks!

**Comment by jhamman** [comment]
Also, pinging folks at Arviz (@rpgoldman, @aloctavodia, @OriolAbril, @ahartikainen), Starfish (@ambrosejcarr, @neuromusic, @joshmoore, @ttung, @sofroniewn), AllenSDK (@jsiegle, @NileGraddis, @tfliss), Sunpy (@wtbarnes, @Cadair), PlasmaPy (@namurphy, @blalterman, @SolarDrew), and Scikit-Learn (@thomasjpfan, @amueller, @adrinjalali). 

It would be great to get feedback from you and Xarray users in your community!

**Reply by OriolAbril** [reply]
Thanks!
